install_scripts/python

```diff
@@ -115,32 +115,6 @@ function check_for_isstreamer {
 }
 
 function download_script {
-	echo "Please select which Initial State app you're using: "
-	echo "1) app.initialstate.com"
-	echo "2) [NEW!] iot.app.initialstate.com"
-	echo -n "Enter choice 1 or 2: "
-	read app_version < /dev/tty
-
-	if [ "$app_version" = "1" ]; then
-		echo -n "Enter app.initialstate.com user email: "
-		read username < /dev/tty
-		echo -n "Enter app.initialstate.com password [input hidden]: "
-		read -s password < /dev/tty
-		echo ""
-
-		if [ -z "$example_location" ]; then
-			example_location="./is_example.py"
-		fi
-
-		python_example=$(curl -X POST "https://api.initialstate.com/api/v1/python/example" -H "X-USER: $username" -H "X-SEC: $password" -H "Accept-Content: text/plain" -m 30 -d "" -s)
-
-		if [ "$python_example" = "INVALID_CREDENTIALS" ]; then
-			echo "invalid credentials, let's try that again!"
-			download_script
-		else
-			echo "$python_example" | $SUDO tee "$example_location" > /dev/null
-		fi
-	else
```

Comment:
Fix the indentation for the body, remove the conditional body closing "fi" to avoid a syntax error.